src/modals/AcceptTermsOfUseModal.jsx

```diff
@@ -0,0 +1,60 @@
+import { useState } from 'react'
+import { Link } from '@react-navigation/native'
+import { useTranslation } from 'react-i18next'
+import {
+  Linking,
+  Text,
+  TouchableOpacity as Button
+} from 'react-native'
+import { CustomModal } from './CustomModal.jsx'
+import { styles } from '../styles.js'
+import { modalStyles } from './modalStyles.js'
+import BouncyCheckbox from "react-native-bouncy-checkbox"
+
+export const AcceptTermsOfUseModal = (props) => {
+  const { t } = useTranslation()
+
+  const [ accepted, setAccepted ]  = useState(false)
+
+  return (
+    <CustomModal
+      modalVisible={props.modalVisible}
+      hideModal={props.hideModal}
+      showCloseButton={props.showCloseButton}
+      title={t('Terms of use {{currentServerName}}', {
+        currentServerName: props.currentServerName
+      })}
+    >
+      <Text style={styles.label}>
+        {t('Please read and accept the terms of use of the server to be able to connect.')}
+      </Text>
+
+      <Text
+        style={styles.link}
+	onPress={async () => { await Linking.openURL(props.termsOfUse.url)}}
+      >
+        {t('Terms of use')}
+      </Text>
+
+      <BouncyCheckbox
+        size={25}
+        fillColor="red"
+        unfillColor="#FFFFFF"
+        text="I've read and accept the terms of use"
+        iconStyle={{ borderColor: "red" }}
+        innerIconStyle={{ borderWidth: 2 }}
+        textStyle={{ fontFamily: "JosefinSans-Regular" }}
+        onPress={setAccepted}
```

Comment:
I just fixed the linting, but I have some questions:

- Why do we change the font?
- Wouldn't it be better to have these styles in a style sheet?
- Is it possible to add the translation to the text?

```suggestion
        size={25}
        fillColor='red'
        unfillColor='#FFFFFF'
        text="I've read and accept the terms of use"
        iconStyle={{ borderColor: 'red' }}
        innerIconStyle={{ borderWidth: 2 }}
        textStyle={{ fontFamily: 'JosefinSans-Regular' }}
        onPress={setAccepted}
```

Reply:
@Haskaris ?

Reply:
Seen with @Haskaris, I removed the custom style and font. Text is now passed through `t()`.